apps/block_scout_web/lib/block_scout_web/templates/stats/index.html.eex

```diff
@@ -1,19 +1,55 @@
 <%= render BlockScoutWeb.Advertisement.TextAdView, "index.html", conn: @conn %>
 <% stats_enabled = Application.get_env(:block_scout_web, :stats_enabled) %>
 <% stats_report_url = Application.get_env(:block_scout_web, :stats_report_url) %>
-<section class="container" data-page="stats-list">
+<% overview_url = BlockScoutWeb.StatsView.get_url(stats_report_url, "overview") %>
```

Comment:
You are converting the same json object from string to a map 5 times to retrieve a single key, and the `get_url` function doesn't actually get a url (it just decodes and fetches a value). 

It's probably not so expensive in this situation, but in general you can decode the object once and fetch the values from there. Either

```elixir
{:ok, urls} = stats_report_url |> Jason.decode()

# .....

<a href="<%= Map.get(urls, "cStables", "default url") %>" target="myiframe">

```

or directly destructure the map

```elixir
{:ok, urls} = stats_report_url |> Jason.decode()

%{"overview" => overview_url, "cStables" => cstables_url ...... } = urls

```